dotorg/dotorg-open-links-new-tab.js

```diff
@@ -0,0 +1,22 @@
+// ==UserScript==
+// @name         Opens all links in new tab
+// @namespace    http://tampermonkey.net/
```

Comment:
Adjust the namespace so it's not the default.
Use https if you can.

Reply:
@andrasguseo  adjust it to what? I checked the other scripts and they have tampermonkey.net as the namespace

Reply:
I see gustavo did one and put https://theeventscalendar.com as the namespace so I did that